### DISCUSSION TITLE
Get banner dimensions

### DISCUSSION THREAD

**Discussion by Xioor** [Poing-Studios/Godot-AdMob-Android-iOS#66] [General]
It would be nice if we could get the banner dimensions, similar to this from the kloder-games ones:
https://github.com/kloder-games/godot-admob/blob/f5996ed22dfab457d611bc1f4f32d6305de3dcd2/admob/ios/src/AdmobBanner.h#L23

**Comment by gumaciel** [comment]
Great, i will definitely do that on next updates! Thanks for the suggestion.

**Reply by Xioor** [reply]
Is it possible to also have hide banner?

**Reply by gumaciel** [reply]
I will try to do that, i have not did that due a bug [`Multiple locks on web thread not allowed`](https://stackoverflow.com/questions/8037407/multiple-locks-on-web-thread-not-allowed-please-file-a-bug-crashing-now), this bug only was solved when i destroy by complete the Banner, but this bug is very old and probably it won't happen nowadays i will give a look.

Until this please use this method: `destroy_banner` https://github.com/Poing-Studios/Godot-AdMob-Android-iOS/blob/master/example/admob_api/MobileAds.gd#L68

**Reply by Xioor** [reply]
Thanks that worked! Good enough for me for now :)
